Source/App/HeadlessMain.cpp

```
@@ -360,35 +436,58 @@ static RunMetrics runScenarioProject(const Scenario& sc) {
-    engine.setSampleRate(sc.sampleRate);
-    engine.setBufferConfig(sc.bufferFrames, kChannels);
+    return renderLoadedProject(trackManager, loadRes, sc);
+}
 
-    // Mirror app wiring for metering + routing.
-    auto meterBuffer = std::make_shared<MeterSnapshotBuffer>();
-    engine.setMeterSnapshots(meterBuffer);
-    trackManager->setMeterSnapshots(meterBuffer);
-    if (auto slotMap = trackManager->getChannelSlotMapShared()) {
-        engine.setChannelSlotMap(slotMap);
+static RunMetrics runScenarioRoundTrip(const Scenario& sc) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)